CONTRIBUTING.md

```diff
@@ -576,6 +576,8 @@ We have a linter that checks a few different things:
   the documented syntax is edited to make the documentation more
   readable.  In this case, you may have to edit
   `doc/tools/docgram/common.edit_mlg` to make `doc_gram_rsts` pass.
+  The same applies if you update documented syntax of a Ltac-defined
+  tactic whose syntax in not defined the grammar.
```

Comment:
This is a bit inaccurate and out of place:
- `Tactic Notation` is not part of Ltac (though as you note, it also applies to Ltac)
- The comment should be somewhere in `doc/tools/docgram/README.md`, which describes the `doc_grammar` at length.  It should mention something about editing `common.edit_mlg` even though it's not quite a cookbook change (e.g. `common.edit_mlg` had to refer to `constr` rather than `one_term)`.  Feel free to add a pointer to that document here.

Also, there's a question of which Tactic Notations and Ltac-defined tactics are worth adding to the manual.  There are 152 Tactic Notations defined in `theories`.  If they're really widely used, it makes sense, but documenting every single one, even those that are used infrequently bloats the documentation.  Currently the doc provides no indication of which tactics are used a lot and which are used infrequently, and of course this varies on a per-user basis.

Reply:
Thanks for elaboration!

> e.g. common.edit_mlg had to refer to constr rather than one_term

I don't understand  this tool well enough (yet) to document this, I'll simply revert the remark for now

>  Feel free to add a pointer to that document here.

It is already here, just a line below :)